aplt/tests/test_logging.py

```diff
@@ -24,7 +24,12 @@ def test_init(self):
         assert obj._output == sys.stdout
         assert obj.format_event == obj.json_format
 
-        filename = os.tempnam()
+        try:
+            filename = os.tempnam()
+        except RuntimeWarning:
```

Comment:
this won't really silence the warning. If these warnings were set to 'error it would indeed catch+silence it, but I doubt anyone's setting it that way. You'd have to make an obscure warning filter to really do this.

Switch this over to tempfile.NamedTemporaryFile() instead, that should be pretty easy and take care of the warning

Reply:
Odd. when running tests, it always displayed the warning. Thanks for the tip about `tempfile`.